cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslReferenceContent.groovy

```diff
@@ -12,47 +12,64 @@ Snippetizer snippetizer = my;
 def l = namespace(lib.LayoutTagLib)
 def st = namespace("jelly:stapler")
 
-
-h1(_("Steps"))
-for (StepDescriptor d : snippetizer.getStepDescriptors(false)) {
-    generateStepHelp(d);
+st.adjunct(includes: 'org.jenkinsci.plugins.workflow.cps.Snippetizer.css.workflow')
+
+h1(_("DSL Reference"))
+div(class:'steps-box basic'){
+	h2(_("Steps"))
```

Comment:
This file was previously formatted with 2 spaces and no tabs, follow the previous format please.